test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -407,17 +407,12 @@ describe('View existing record,', function() {
                     expect(addRelatedRecordLink.getText()).toBe("Add", "The Add button is not displayed as Add");
 
                     addRelatedRecordLink.click().then(function(){
```

Comment:
Add `chaisePage.waitForElement(chaisePage.recordEditPage.getModalTitle());` after this line.